src/storage/test/TestUtils.h

```diff
@@ -74,7 +74,31 @@ class TestUtils {
                                                             localhost,
                                                             workers);
         store->init();
-        sleep(1);
+
+        // When useMetaServer is true, we deal with it using waitUntilAllElected.
+        if (useMetaServer) {
+            sleep(1);
```

Comment:
`sleep(1)` here?

Reply:
Yeah, when useMetaServer is true, follow the original logic

Reply:
Yes, I think it's useless.

Reply:
I think  here `sleep`, and the  following `waitUntilAllElected `  basically does not need to `sleep`  anymore.

Reply:
This pr is not needed.